tickets/src/components/interface/EventInfo.tsx

```diff
@@ -9,7 +9,11 @@ export const EventInfo = ({ event }: any) => {
     <div>
       {event.image && <Banner src={event.image} />}
       <Title>{event.name}</Title>
-      <EventDate date={event.date} duration={event.duration} />
+      {event.name === 'EthWaterloo' ? (
```

Comment:
How would you feel about doing the comparison against a normalized version to avoid any weirdness